app/src/main/java/com/mapbox/maps/testapp/examples/localization/LocalizationSettingsActivity.kt

```diff
@@ -0,0 +1,308 @@
+package com.mapbox.maps.testapp.examples.localization
+
+import android.content.Context
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.bindgen.Expected
+import com.mapbox.bindgen.None
+import com.mapbox.common.SettingsServiceFactory
+import com.mapbox.common.SettingsServiceInterface
+import com.mapbox.common.SettingsServiceStorageType
+import com.mapbox.maps.extension.style.sources.generated.vectorSource
+import com.mapbox.bindgen.Value
+import com.mapbox.maps.MapboxMap
+import com.mapbox.maps.extension.observable.eventdata.MapLoadingErrorEventData
+import com.mapbox.maps.extension.observable.model.MapLoadErrorType
+import com.mapbox.maps.extension.style.sources.addSource
+import com.mapbox.maps.plugin.delegates.listeners.OnMapLoadErrorListener
+import com.mapbox.maps.testapp.R
+import com.mapbox.maps.testapp.databinding.ActivityLocalizationSettingBinding
+import java.util.*
+import com.mapbox.common.MapboxCommonSettings.LANGUAGE
+import com.mapbox.common.MapboxCommonSettings.WORLDVIEW
+import com.mapbox.maps.Style
+import kotlinx.coroutines.*
+import kotlinx.coroutines.launch
+import kotlin.coroutines.CoroutineContext
+
+/**
+ * An example to shows how to set Language and Worldview settings with new settings api.
```

Comment:
I would suggest refraining from wording as "new" since that won't stand the test against time very well. 
While it's now new, it won't be in a couple months from now.